var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -54,7 +54,8 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("blas")
     depends_on("lapack")
     depends_on("mpi", when="+mpi")
-    depends_on("libxsmm@1.11:~header-only", when="smm=libxsmm")
+
+    depends_on("libxsmm@:1~header-only", when="smm=libxsmm")
```

Comment:
Why drop the lower compat bound?

Reply:
it should be fixed now. No particular reason for this drop.